README.md

```diff
@@ -19,6 +19,10 @@ This package offers a plugin for the Eclipse to use the code analysis software [
 mvn clean verify
 ```
 
+## Usage
+To learn how to use the plug-in please refer to the [Using the plug in]
+(https://github.com/coala-analyzer/coala-eclipse/wiki/Using-the-plug-in) wiki page.
```

Comment:
are you sure? wikis are often outdated and never read first. i think you should rather explain in here briefly and give more explanations there